src/models/Repos.js

```diff
@@ -10,6 +10,7 @@ function addToArray(value) {
 }
 
 function filterRelevantFields(repos) {
+  console.log(repos);
```

Comment:
i must have pulled down new changes because I no longer have this in my codebase